metadata/src/test/java/org/apache/kafka/image/ClusterImageTest.java

```diff
@@ -88,7 +101,7 @@ public class ClusterImageTest {
             setId(2).
             setEpoch(123).
             setIncarnationId(Uuid.fromString("hr4TVh3YQiu3p16Awkka6w")).
-            setListeners(Arrays.asList(new Endpoint("PLAINTEXT", SecurityProtocol.PLAINTEXT, "localhost", 9093))).
+            setListeners(Arrays.asList(new Endpoint("PLAINTEXT", SecurityProtocol.PLAINTEXT, "localhost", 9094))).
```

Comment:
`ReplicaManagerTest` reuse the `IMAGE1` to test the listeners, hence this change break some test cases. Do you have free cycle to fix it? If not, I can file PR to fix it :)

Reply:
I have filed a ticket https://issues.apache.org/jira/browse/KAFKA-16447

please feel free to take over it

Reply:
I just found this as well. I opened https://issues.apache.org/jira/browse/KAFKA-16451 but will close as a duplicate

Reply:
Responded on https://issues.apache.org/jira/browse/KAFKA-16447, someone is volunteering to fix, will check in again tomorrow!